cpp/spherical_volume_rendering_util.cpp

```diff
@@ -102,7 +102,27 @@ inline int calculateAngularVoxelIDFromPoints(
     const double d3 = (X_diff * X_diff) + (Y_diff * Y_diff);
     if (d1d2 < d3 || svr::isEqual(d1d2, d3)) return i;
   }
-  return i;
+  return angular_max.size() + 1;
+}
+
+// Returns true if the "step" taken from the current voxel ID remains in
+// the grid bounds.
+inline int inBoundsAzimuthal(const SphericalVoxelGrid &grid,
+                    const int step,
+                    const int azi_voxel) noexcept {
+  const double radian = (azi_voxel + 1)*grid.deltaPhi();
+  const double angval = radian - abs(step * grid.deltaPhi());
+  return angval <= grid.sphereMaxBoundAzi() && angval >= grid.sphereMinBoundAzi() ;
+}
+
+// Returns true if the "step" taken from the current voxel ID remains in
+// the grid bounds.
+inline int inBoundsPolar(const SphericalVoxelGrid &grid,
+                         const int step,
+                         const int pol_voxel) noexcept {
+  const double radian = (pol_voxel + 1)*grid.deltaTheta();
+  const double angval = radian - abs(step * grid.deltaTheta());
```

Comment:
Change abs -> std::abs please. This avoids namespace collisions (Same case for inBoundsAzimuthal).